src/families/hedera/utils.ts

```diff
@@ -0,0 +1,29 @@
+import BigNumber from "bignumber.js";
+import { Account } from "../../types";
+import { Transaction } from "./types";
+
+export function calculateAmount({
+  account,
+  transaction,
+}: {
+  account: Account;
+  transaction: Transaction;
+}): { amount: BigNumber; estimatedFees: BigNumber; totalSpent: BigNumber } {
+  // NOTE: Hedera declares stable fees in USD
+  //       If we can get the current USD/HBAR price here..
```

Comment:
Question for @gre: do you think we can get the countervalue here?

Reply:
As discussed during our sync meeting, please keep displaying fees in coin unit, as Ledger Live isn't designed to display fees in fiat unit.

Reply:
👍🏻